realm/src/main/java/io/realm/Realm.java

```diff
@@ -163,19 +163,18 @@ protected void finalize() throws Throwable {
     }
 
     /**
-     * Realm static constructor for the default Realm "default.realm".
-     * {@link #close()} must be called when you are done using the Realm instance.
-     * <p>
-     * It sets auto-refresh on if the current thread has a Looper, off otherwise.
-     *
+     * Realm static constructor for the default Realm file "default.realm".
```

Comment:
My build environment(of course Gradle) can generate Javadoc as expected.
What's the differences...

I have no idea because `[@value foo}` is not a new notation...

Can I see the error message?